packages/plugins/rubygem-smart_proxy_monitoring/rubygem-smart_proxy_monitoring.spec

```diff
@@ -98,26 +69,27 @@ mv %{buildroot}%{gem_instdir}/settings.d/monitoring_icinga2.yml.example \
 mv %{buildroot}%{gem_instdir}/settings.d/monitoring_icingadirector.yml.example \
    %{buildroot}%{foreman_proxy_settingsd_dir}/monitoring_icingadirector.yml
 
-# additional config
-mkdir -p %{buildroot}%{_root_sysconfdir}/foreman-proxy/monitoring
```

Comment:
I think we should keep that (in `_sysconfdir`), as it's the folder the examples place certificates in etc.

Reply:
So you mean keep it change it from
`%{buildroot}%{_root_sysconfdir}/foreman-proxy/monitoring`
to
`%{buildroot}%{_sysconfdir}/foreman-proxy/monitoring`

Reply:
Yepp! And keep the `%dir` entry in `%files` accordingly.